CMakeLists.txt

```diff
@@ -91,7 +93,7 @@ set(CPACK_DEBIAN_PACKAGE_DEPENDS "ibus (>= 1.5.1)")
 set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
 set(CPACK_RPM_PACKAGE_LICENSE "GPLv3")
 set(CPACK_RPM_PACKAGE_URL "https://openbangla.github.io/")
-set(CPACK_RPM_PACKAGE_REQUIRES "qt5-qtbase >= 5.9.0, ibus >= 1.5.1, ibus-libs >= 1.5.1")
+set(CPACK_RPM_PACKAGE_REQUIRES "qt5-qtbase >= 5.9.0, ibus >= 1.5.1, ibus-libs >= 1.5.1, zstd >= 1.3.3")
```

Comment:
the zstd binaries are not required. this should be `libzstd`, right?
also, shouldn't `CPACK_DEBIAN_PACKAGE_DEPENDS` list qt5 and zstd dependencies?

Reply:
> the zstd binaries are not required. this should be `libzstd`, right?
Yeah!
> also, shouldn't `CPACK_DEBIAN_PACKAGE_DEPENDS` list qt5 and zstd dependencies?
Actually the dependencies are automatically included by `dpkg-shlibdeps` as we set `CPACK_DEBIAN_PACKAGE_SHLIBDEPS`.

Reply:
ah. okay then. other changes are alright.